src/components/auth/Congratulations.js

```diff
@@ -1,8 +1,7 @@
 import React from "react";
```

Comment:
This was actually me just experimenting with images and media in the public folder.